### DISCUSSION TITLE
BTT Pad 7

### DISCUSSION THREAD

**Discussion by chainz316** [LeeOtts/Ender3v2-Klipper-Configs#4] [General]
I'm fairly new to klipper. Used to use octoprint on my pi but changed that out for the BTT Pad 7.
Ender 3 v2, 4.2.2 board, CR Touch, default extruder. 
Will all these files work fine or anything I need to change in the printer.cfg as this file is made for a pi?
I know I'll have to change the part where it uses the 4.2.2 board instead of 4.2.7.

**Comment by LeeOtts** [comment]
That is a good question. 

I am sure some things will be different like the MCU and the pins for the 4.2.2 and 4.2.7 are different. 

I would install klipper on your BTT make the changes you can. Figure out the mcu and pins for the 4.2.2 and try it. 

The web interface will let you know why it fails with an error message.  So take the error and see what google says. Make the change and see if that fixes it or not, then on to the next error, then back to google and so on. 

If you can't get it working respond here with the errors and I will try to help you out. 

Let me know if you get it working. I'm interested in your results. Your experiences could help someone else down the line as well.
